new/LIT6802SobaAshafaC_.xml

```diff
@@ -0,0 +1,77 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6802SobaAshafaC_" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1"> ወሶበ አጽሐፈ ደቅስዮስ መጽሐፈ ተአምርኪ ቡሩክ ፨</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Wa-soba ʾaṣḥafa Daqsǝyos maṣḥafa taʾammǝrǝki buruk</title>
+                <title xml:lang="en" corresp="#t1" type="normalized">When Daqsǝyos let the blessed book of  your miracles be written</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="GS"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+               
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Apocrypha"/>
+                    <term key="Chants"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="GS" when="2023-01-05">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="ecrm:CLP46i_may_form_part_of" active="LIT6802SobaAshafaC_" passive="LIT5673AkkonuColl"/>
+                </listRelation>
+                <div type="edition">
+                    <div type="textpart" xml:lang="gez">
+                        <note>The following text is copied from <ref type="mss" corresp="GG-0044"/>.</note>
```

Comment:
Maybe it could be clarified in this note that this is the entire text, not only a part of the poem.